imports/plugins/core/orders/client/components/orderSearch.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { compose } from "recompose";
+import PropTypes from "prop-types";
+import { Components, withPermissions } from "@reactioncommerce/reaction-components";
+
+/**
+ * Class representing the Products React component
```

Comment:
This represents the Products React component? Is this a copy/paste error? `load more products callback`?

Reply:
tfw you copy directly from the React docs but forget to change them 😈